exporter/instanaexporter/exporter.go

```diff
@@ -111,6 +112,20 @@ func (e *instanaExporter) export(ctx context.Context, url string, header map[str
 		return consumererror.NewPermanent(err)
 	}
 
+	tlsConf, err := e.config.TLSSetting.LoadTLSConfig()
+
+	if err != nil {
+		return consumererror.NewPermanent(err)
+	}
+
+	if tlsConf.RootCAs != nil {
```

Comment:
You shouldn't need to do this here, as you are using the ToClient from the `confighttp`. Is this logic missing there? Perhaps it should be around this part?

https://github.com/open-telemetry/opentelemetry-collector/blob/c9149646002bfbc74744113cd14ca4f3edaa290c/config/confighttp/confighttp.go#L134-L140

Reply:
Good call! I tested the app without this code and it just works as intended :)
Thanks!